CHANGELOG.md

```diff
@@ -5,7 +5,19 @@ All notable changes to this project will be documented in this file.
 The format follows [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.6.2] - 2020-06-09
+## [0.6.5] - 2020-06-11
+
+### Changed
+
+-   Create plugin directory if it doesn't exist before downloading plugins
+
+## [0.6.4] - 2020-06-09
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/amplify-education/terrawrap/pullRequest?prid=5677852)